### DISCUSSION TITLE
Permissions to view Rack Role

### DISCUSSION THREAD

**Discussion by tkcg2k** [netbox-community/netbox#13758] [Help Wanted!]
Hi, is there a way to config permission by rack role?

we using rack per team (system, storage, network...).

thx

**Comment by candlerb** [comment]
Yes and no.

Clearly you can do this for the rack itself.

For objects which relate directly to a rack (e.g. Device) you can set permissions on `rack__role`. Similarly if this can be obtained via a parent object, for example interfaces could match on `device__rack__role`.

But for things with an indistinct relation to a rack, like cables or IP addresses, it will be hard.
